plugins/modules/swim_intent.py

```diff
@@ -717,6 +728,7 @@ def get_have(self):
                 have["distribution_image_id"] = self.have.get("imported_image_id")
 
             else:
+                self.log("Image details for distribution not provided", "CRITICAL")
```

Comment:
self.log("Image details required for distribution have not been provided", "CRITICAL")

Reply:
added.